### ISSUE TITLE
[FEATURE]  Pass dynamic values to override config chatflow tool

### ISSUE THREAD

**Issue by vitusan** [FlowiseAI/Flowise#3385]
**Describe the feature you'd like**
It will be nice to be able to use variables and flow data into the override config of the chatflow tool. 

**Additional context**
My use case is to customize the metadata filter from the pinecone index, witch is possible via API, but not via variables and flow data, so i thought of using the chatflow tool to override the pinecone configuration, but sadly, the values here are not dynamic and must be hardcoded and i need to extract this metadata from user interaction via LLM parsing. 
![image](https://github.com/user-attachments/assets/80353a82-f41e-45c0-a1ab-6c1514dfba72)

**Comment by HenryHengZJ** [comment]
you should be able to do like following no?
```
day: {{$vars.day}}
```

**Comment by vitusan** [comment]
Yeah, but i'm using state for each chat, because it's extracted from the user's input. Tried the following but didn't work:

![image](https://github.com/user-attachments/assets/c749c918-c1a1-42da-9a11-123761674c19)

**Comment by niztal** [comment]
@vitusan @HenryHengZJ 

Are there any updates regarding this enhancement or maybe there's another solution rather than using a custom tool for such functionality?

Thanks

**Comment by HenryHengZJ** [comment]
fix: https://github.com/FlowiseAI/Flowise/pull/5203
